ui/lib/components/accordion/accordion.dart

```diff
@@ -126,24 +161,87 @@ class Accordion {
     return _accordionItems.firstWhere((item) => item._id == id, orElse: () => null);
   }
 
+  void _makeSortable() {
+    if (_sortableId == null) return;
+
+    _accordionItems.forEach((accordionItem) => accordionItem.sortableId = _sortableId);
+    var accordionItemRenderElements = _accordionItems.map((accordionItem) => accordionItem.renderElement).toList();
+
+    _draggable?.destroy();
+    _dropzone?.destroy();
+    
+    _draggable = Draggable(accordionItemRenderElements, avatarHandler: AvatarHandler.clone(), verticalOnly: true, handle: '.$ACCORDION_ITEM_HANDLE_CSS_CLASSNAME');
+    _draggable.onDragStart.listen((DraggableEvent event) {
+      collapseAllItems();
+    });
+
+    _dropzone = Dropzone(accordionItemRenderElements);
+    _dropzone.onDragOver.listen((DropzoneEvent event) {
+      event.avatarHandler.avatar?.classes?.toggle('drag-element', true);
+    });
+    _dropzone.onDragEnter.listen((event) {
+      if (_dragAllowed(event)) {
+        event.dropzoneElement.classes.toggle('drag--over', true);
+      }
```

Comment:
nit: let's switch to using an early exist strategy as you do fr onDrop:
```suggestion
      if (!_dragAllowed(event)) return;
      event.dropzoneElement.classes.toggle('drag--over', true);
```